docs/deploying-airbyte/local-deployment.md

```diff
@@ -7,17 +7,18 @@ These instructions have been tested on MacOS, Windows 10 and Ubuntu 20.04.
 
 ## Setup & launch Airbyte
 
-* Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
-* After Docker is installed, you can immediately get started locally by running:
+- Install Docker on your workstation \(see [instructions](https://www.docker.com/products/docker-desktop)\). Make sure you're on the latest version of `docker-compose`.
+- After Docker is installed, you can immediately get started locally by running:
 
 ```bash
 git clone https://github.com/airbytehq/airbyte.git
 cd airbyte
 docker-compose up
 ```
 
-* In your browser, just visit [http://localhost:8000](http://localhost:8000)
-* Start moving some data!
+- In your browser, just visit [http://localhost:8000](http://localhost:8000)
+- You will be asked for a username and password. By default, that's username `airbyte` and password `password`. Once you deploy airbyte to your servers, be sure to change these.
+- Start moving some data!
```

Comment:
should this link to the security Airbyte doc so it's obvious this is something we can easily change?